app/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma": "none"
```

Comment:
@jamesmockett There is a Prettier pre-commit hook but no prettier config file so added one as the trailing comma was giving me issues despite changing my settings in VSCode.

Reply:
This has been annoying me too as it's relying on the default config that comes with Prettier. I'm not sure why it's not using the standard Guardian Prettier and ESLint configs, but I'm looking at adding those and cleaning up the entire codebase so files don't get reformatted every time you touch something.